core/src/main/java/hudson/security/GlobalSecurityConfiguration.java

```diff
@@ -72,6 +72,10 @@ public int getSlaveAgentPort() {
         return Jenkins.getInstance().getSlaveAgentPort();
     }
 
+    public boolean isSlaveAgentPortEnforced() {
```

Comment:
Nitpicking territory: We're not actually using `XXX`, at least not for `@since`, it's always `TODO`. Only change if there are further changes necessary anyway.